.github/workflows/build.yml

```
@@ -84,3 +110,15 @@ jobs:
           elif [ "${{ matrix.lang }}" == "dotnet" ]; then
             docker run --rm ${{ env.MAIN_IMAGE_FULL_NAME }} dotnet --version
           fi
+
+      - name: ${{ matrix.lang }}-fips sanity test
+        run: |
+          if [ "${{ matrix.lang }}" == "javascript" ]; then
+            docker run --rm ${{ env.MAIN_IMAGE_FULL_NAME }}-fips node --version
+          elif [ "${{ matrix.lang }}" == "python" ]; then
+            docker run --rm ${{ env.MAIN_IMAGE_FULL_NAME }}-fips python --version
+          elif [ "${{ matrix.lang }}" == "golang" ]; then
+            docker run --rm ${{ env.MAIN_IMAGE_FULL_NAME }}-fips go version
+          elif [ "${{ matrix.lang }}" == "dotnet" ]; then
+            docker run --rm ${{ env.MAIN_IMAGE_FULL_NAME }}-fips dotnet --version
+          fi
```

Maybe a fips sanity check as well, juuussttt in case

```suggestion
          fi
          docker run --rm ${{ env.MAIN_IMAGE_FULL_NAME }}-fips sh -c '[ -n "$OPENSSL_FIPS" ] || exit 1'
```

---

sweet